.gitignore

```diff
@@ -0,0 +1 @@
+tox.ini
```

Comment:
@rgil-splunk I believe `tox.ini` should be uploaded, right?
Please see [MaxMind example](https://github.com/splunk-soar-connectors/maxmind/blob/next/tox.ini)

Reply:
I have made the suggested code changes.